manager-bundle/src/ContaoManager/Plugin.php

```diff
@@ -407,6 +408,52 @@ private function addDefaultDoctrineMapping(array $extensionConfigs, ContainerBui
         return $extensionConfigs;
     }
 
+    /**
+     * Adds a default ORM mapping for the App namespace if none is configured.
+     *
+     * @return array<string,array<string,array<string,array<string,mixed>>>>
+     */
+    private function enableStrictMode(array $extensionConfigs): array
+    {
+        $driver = '';
+        $options = [];
+
+        foreach ($extensionConfigs as $config) {
+            if (null !== ($driverConfig = $config['dbal']['connections']['default']['driver'] ?? null)) {
+                $driver = $driverConfig;
+            }
```

Comment:
We also need to handle the `url` key I think, see line 314

Reply:
Adjusted in fb2cd9e5b89e7ab388f80cc28e3cd9f03089460d - could you have another look?